.travis.yml

```diff
@@ -1,5 +1,7 @@
 dist: trusty
 sudo: false
+addons:
```

Comment:
should probably use fix/chrome_startup coding in travis ... so this can be removed